roles/container-engine/docker/tasks/reset.yml

```diff
@@ -1,4 +1,70 @@
 ---
+
+- name: Docker | Get package facts
+  package_facts:
+    manager: auto
+  tags:
+    - reset_docker
+
+- name: Docker | Find docker packages
+  set_fact:
+    docker_packages_list: "{{ ansible_facts.packages.keys() | select('search', '^docker*') }}"
+    containerd_package: "{{ ansible_facts.packages.keys() | select('search', '^containerd*') }}"
+  tags:
+    - reset_docker
+
+- name: Docker | Stop all running container
+  shell: "{{ docker_bin_dir }}/docker kill $({{ docker_bin_dir }}/docker ps -q)"
+  args:
+    executable: /bin/bash
+  register: remove_all_containers
+  retries: 5
+  until: remove_all_containers.rc == 0
+  changed_when: true
+  delay: 5
+  ignore_errors: true # noqa ignore-errors
```

Comment:
ansible-lint shows the error here
```
roles/container-engine/docker/tasks/reset.yml:25: yaml too few spaces before comment (comments)
```
Please 2 spaces before `#` like the other lines.